files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -24,17 +24,25 @@ set-kargs-hardening:
       --append-if-missing="l1d_flush=on" \
       --append-if-missing="gather_data_sampling=force" 
     echo "Hardening kargs set."
-
-# Add additional (unstable) boot parameters for hardening (requires reboot)
-set-kargs-hardening-unstable:
-    #!/usr/bin/pkexec /usr/bin/bash
-    echo "Warning: setting these kargs may lead to boot issues on some hardware."
+    echo "Would you like to set additional (unstable) hardening kargs? [y/N] (Warning: setting these kargs may lead to boot issues on some hardware.)"
```

Comment:
one optimization might be to ask the questions first, and then assemble the command afterwards, otherwise the user may have to wait up to three times for kargs to set